config/locales/en.yml

```diff
@@ -79,6 +79,29 @@ en:
   update: "Update"
   updated: "Updated"
 
+  general:
```

Comment:
These translations and whether or not to nest will need to be addressed in another branch.